src/design-system/HelpButtonWithTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+import Colors from "./Colors";
+import Tooltip from "./Tooltip";
+
+const HelpButton = styled.span`
```

Comment:
A style note: in the mocks, this is always a little bit separate from the text label that it relates to, but in the current implementation it follows very closely from the text. Maybe just some left padding would suffice, or having it right-align to its container?

Reply:
Padding is not finalized at all yet

Reply:
I'll merge this PR now, but we'll track this in #46.